fixtures/signer-config.json

```diff
@@ -0,0 +1,16 @@
+{
+	"server": {
+		"http_addr": ":4444",
+		"grpc_addr": ":7899",
+		"tls_cert_file": "./notary-signer.crt",
+		"tls_key_file": "./notary-signer.key",
+		"client_ca_file": "./notary-server.crt"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "mysql",
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
Lets finally update the username/password :-P

It just needs to match whatever we have set here: https://github.com/docker/notary/blob/master/notarymysql/start

For consistency with a default mysql development installation, let's set `root@tcp(notarymysql:3306)/notary` (by default mysql only listens on localhost which is why it doesn't complain too much about the root user with no password during installation).